pkg/operator/k8sutil/pod.go

```diff
@@ -397,3 +397,31 @@ func removeDuplicateEnvVarsFromContainer(container *v1.Container) {
 	}
 	container.Env = vars
 }
+
+func IsPodScheduled(clientSet kubernetes.Interface, namespace, selector string) (bool, error) {
+	podList, err := ListPodWithLabels(clientSet, namespace, selector)
+	if err != nil {
+		return false, errors.Wrap(err, "failed to check if pod is scheduled")
+	}
+
+	podNode := podList.Items[0].Spec.NodeName
+	if podNode == "" {
```

Comment:
nit: check for it directly instead of using a var
```suggestion
	if podList.Items[0].Spec.NodeName == "" {
```

Reply:
done